api/ocm/extensions/download/handlers/helm/registration.go

```diff
@@ -0,0 +1,74 @@
+package helm
+
+import (
+	"fmt"
+	helmregistry "helm.sh/helm/v3/pkg/registry"
+
+	"github.com/mandelsoft/goutils/errors"
+	"golang.org/x/exp/slices"
+
+	"ocm.software/ocm/api/oci/artdesc"
+	"ocm.software/ocm/api/oci/extensions/repositories/artifactset"
+	"ocm.software/ocm/api/ocm/cpi"
+	downloadhandlers "ocm.software/ocm/api/ocm/extensions/download"
+	"ocm.software/ocm/api/utils/listformat"
+	"ocm.software/ocm/api/utils/registrations"
+)
+
+const PATH = "helm/artifact"
```

Comment:
I hope we don't lose control for so many different `const` values...
https://github.com/open-component-model/ocm/blob/main/cmds/ocm/commands/ocmcmds/common/inputs/types/helm/type.go#L7
https://github.com/open-component-model/ocm/blob/main/api/ocm/elements/artifactaccess/helmaccess/resource.go#L12
https://github.com/open-component-model/ocm/blob/main/api/ocm/extensions/artifacttypes/const.go#L16